renovate.json

```diff
@@ -13,18 +13,6 @@
       "extends": [
         ":separateMajorReleases"
       ]
-    },
```

Comment:
I've updated xunit in the previous PR, so this rule is not needed anymore

Reply:
Updating xunit worked with the analyzers test package? I think that's the issue we were trying to avoid with this rule?

Reply:
That's an issue only when people don't read the doc ;)

https://github.com/dotnet/roslyn-sdk/blob/e9aff578fa3d6d30a1e753dbaa604f87f312ebf0/src/Microsoft.CodeAnalysis.Testing/README.md#obsolete-packages